packages/orchestration/src/exos/chain-hub.js

```diff
@@ -22,6 +26,15 @@ import { CosmosChainInfoShape, IBCConnectionInfoShape } from '../typeGuards.js';
  *   : ChainInfo} ActualChainInfo
  */
 
+/**
+ * @typedef {object} DenomDetail
+ * @property {string} baseName - name of issuing chain; e.g. cosmoshub
+ * @property {Denom} baseDenom - e.g. uatom
+ * @property {string} chainName - name of holding chain; e.g. agoric
+ * @property {Brand} [brand] - vbank brand, if registered
+ * @see {ChainHub} `registerAsset` method
+ */
```

Comment:
this works it's a bit circular. since this type is used only for the guard, shall we keep it with the guard? That would suggest moving this to the `typeGuards.js` module, but I notice this shape is used only in this exo so we shouldn't export it. Instead please move the guard here.

Another reason that's better than moving the type out is so the properties descriptions are easy to access. Looking it more I see that TS LSP isn't picking up the the jsdoc. 
<img width="381" alt="Screenshot 2024-07-25 at 11 17 13 AM" src="https://github.com/user-attachments/assets/67fb5a46-9b41-49d9-966f-66f3652e4240">

I tried changing to `.ts syntax` and that doesn't solve it.

Reply:
ok; done